cdap-ui/app/cdap/components/Reports/Customizer/StatusSelector/StatusPopover.js

```diff
@@ -60,16 +53,7 @@ class StatusPopoverView extends Component {
       newArr.splice(index, 1);
     }
 
-    this.setState({
-      selections: newArr
-    });
-  };
-
-  apply = () => {
-    this.props.onApply(this.state.selections);
-
-    // Closing popover
-    document.body.click();
+    this.props.onApply(newArr);
```

Comment:
I think with this change, you can remove the state `selections` and just do everything you need from props